operations/hpe_pdu/hpe_pdu_admin_procedures.md

```diff
@@ -0,0 +1,119 @@
+## HPE PDU Admin Procedures
+
+The following procedures are used to manage the HPE Power Distribution Unit (PDU):
+
+* [Connect to HPE PDU Web Interface](#connect-to-hpe-pdu-web-interface)
+* [HPE PDU Initial Set-up](#hpe-pdu-initial-set-up)
+* [Update HPE PDU Firmware](#update-hpe-pdu-firmware)
+* [Change HPE PDU User Passwords](#change-hpe-pdu-user-passwords)
+* [Discover HPE PDU after upgrading CSM](#discover-hpe-pdu-after-upgrading-csm)
+
+**Because of the polling method used to process sensor data from the HPE PDU, telemetry data may take up to 6 minutes to refresh; this includes outlet status reported by HSM.**
+
+
+### Connect to HPE PDU Web Interface
+The following is needed before running this procedure:
+* IP address or Domain Name of ncn-m001
+* Xname of the HPE PDU
+* Root password for ncn-m001
+* Admin password for HPE PDU _(default: 12345678)_
+
+1. Use the `ssh` command from your local machine:
+    ```bash
+    > ssh -L 8443:{PDU_xname}:443 -N root@{ncn-m001_ip}
+    ```
+    Where `{PDU_xname}` is the xname of the PDU and `{ncn-m001_ip}` is the ip address of ncn-m001.\
+    You will need to enter the root password for ncn-m001.
+
+2. Connect to: [`https://localhost:8443`](https://localhost:8443) using a web browser.
+
+3. Log in with the `admin` username.  Enter the admin password.  If you have not changed the admin password, there will be a prompt to change the password.
+
+### HPE PDU Initial Set-up
+Use this procedure to do the following:
+* Ensure redfish is enabled
+* Add the default user
+* Enable Outlet Control
+
+1. Connect to the HPE PDU Web Interface (See [Connect to HPE PDU Web Interface](#connect-to-hpe-pdu-web-interface)) and log in as `admin`.
+
+#### Ensure Redfish is Enabled
+1. Using the **"Settings"** Icon (gear in computer monitor in top right corner) navigate to **"Network Settings"**.

```

Comment:
Icon can be lower case in both of these steps.

Reply:
Updated